lib/logic.rb

```diff
@@ -0,0 +1,40 @@
+class TicTacToe
+  attr_accessor :new_array
+  attr_reader :win_array
+  attr_accessor :win_variable
```

Comment:
- [ ] Use attr_readers/attr_writers instead of attr_accessors. The student must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything. <TODO: explain why> ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- Your project still works even if the first attr_accessor is commented and the two remaining are attr_reader so lets fix it!